mne/decoding/time_gen.py

```diff
@@ -256,7 +267,7 @@ def f(x):
         self.estimators_ = sum(out, list())
         return self
 
-    def predict(self, epochs, test_times=None, picks=None):
+    def predict(self, epochs, test_times=None, y=None, picks=None):
```

Comment:
gat.fit(epochs[:10]).score(epochs[:10])

should be the way to go. Don't by pass the fit method...